src/main/java/frc/robot/subsystems/Input.java

```diff
@@ -62,5 +64,9 @@ public static boolean getDecPID() {
   public static boolean togglePIDTuning() {
     return mRotJoystick.getTriggerReleased();
   }
+
+  public static boolean getLeftBumperXbox() {
```

Comment:
Make this describe what the button is mapped to rather than the button itself(ie: intake rather than left bumper xbox) so that its easier to see the controls